lib/cylc/scheduler_cli.py

```diff
@@ -66,18 +68,25 @@ def main(is_restart=False):
     """CLI main."""
     options, args = parse_commandline(is_restart)
 
+    # Check suite is not already running before start of host selection.
+    try:
+        SuiteSrvFilesManager().detect_old_contact_file(args[0])
+    except SuiteServiceFileError as exc:
+        sys.exit(exc)
```

Comment:
We can remove this logic from `cylc.scheduler` (there is no need to check twice).

Reply:
Good point, though thinking about it, in rare cases wouldn't it be possible for a suite of the same name to start after this check, say during a host selection over many hosts which I have observed to take an order of magnitude of seconds, or during start-up? In that case it might be safest to keep the supplementary check in Scheduler.

Reply:
Well, yes I guess, though this is true at any point of the start up process, reading in the suite configuration can take in the order of tens of seconds...

Reply:
Poke

Reply:
I think it is OK to leave this for now. It is cheap enough to do this check.

Reply:
I'd rather not leave duplicate code lying about, we won't get around to removing it later.